cli/args/flags.rs

```diff
@@ -2896,7 +2896,7 @@ static ALLOW_WRITE_HELP: &str = concat!(
 
 static DENY_WRITE_HELP: &str = concat!(
   "Deny file system write access. Optionally specify denied paths.\n",
-  "Docs: https://deno.land/manual@v",
+  "Docs: https://deno.land/manual",
   env!("CARGO_PKG_VERSION"),
```

Comment:
I think we need to remove all of these `env!("CARGO_PKG_VERSION)` otherwise the string will read something like:
```
Docs: https://deno.land/manual1.44.2/basics/permissions
```

Reply:
Hello @bartlomieju 
Fixed =)